spec/index.html

```diff
@@ -1466,164 +1473,233 @@ <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
 
     <p>This limitation should not be an issue in practice.
-      The special graph name `rdf:unstarMapping` is unlikely to be in used in any published dataset,
-      as it was not defined in the RDF namespace prior to this specification.
+      The `rdf:TripleTerm` type is unlikely to be in used in any published graph or dataset,
+      as it was not defined prior to this specification.
       For this reason, using it would actually have been bad practice.
-      As for future datasets, their authors should consider the graph name `rdf:unstarMetadata` to be reserved, in order to prevent interference with the `unstar` algorithm.
+      As for future graphs and datasets, their authors should consider this type to be reserved, in order to prevent interference with the [=classicize=] transformation.
     </p>
 
-    <p>Another consequence of this restriction is that users should be careful when merging dataset in an application that makes use of the `unstar` algorithm.
-      More precisely, merging a [=Full=] <a>RDF dataset</a> (containing at least one <a>triple term</a>)
-      with a [=Classic=] <a>RDF dataset</a> resulting from the application of `unstar` (and therefore containing an `rdf:unstarMetadata` named graph)
-      would result in a "hybrid" dataset that `unstar` can not process.
-      Such application should make sure to apply `unstar` to every dataset priori to merging them.
-      Since `unstar` is idempotent, there is no harm in applying it more than necessary.</p>
+    <p class=note>
+      This is one of the reasons why this transformation introduces a new vocabulary
+      (`rdf:TripleTerm`, `rdf:ttSubject`, `rdf:ttPredicate`, `rdf::ttObject`),
+      rather than repurposing the existing <a data-cite="RDF12-SCHEMA#ch_reificationvocab">reification vocabulary</a>
+      (`rdf:Statement`, `rdf:subject`, `rdf:predicate`, `rdf:object`).
+      Contrarily to `rdf:TripleTerm`, `rdf:Statement` is known to used in widely used datasets (e.g. <a href="https://www.uniprot.org/">Uniprot</a>),
+      so deprecating its usage as "reserved" was not an option.
+    </p>
 
+    <p>Another consequence of this restriction is that users should be careful when merging graphs in an application that [=classicize=] graphs or datasets.
+      More precisely, merging a [=Full=] [=RDF graph=] (containing at least one [=triple term=])
+      with a [=classicized=] [=RDF graph=] (and therefore potentially containing [=blank node=] instances of `rdf:TripleTerm`)
+      could result in a "hybrid" graph that can not be transformed.
+      Such applications should make sure to [=classicize=] every graph prior to merging them.
+      Conversely, applications supporting RDF [=Full=] should make sure to apply the reverse tranformation to any graph that is known or likely to be [=classicized=],
+      to avoid creating such "hybrid" graphs.
+      Since these transformations are <em>idempotent</em>, there is no harm in applying them more than necessary.
```

Comment:
Its troubling that this caution was written to *users* and then later, to *applications*, rather than *implementers* who ought to bear the burden. Further editing will be good. Also, this segment --
```
      Therefor, such applications should [=classicize=] every graph prior to merging them.
      Conversely, applications supporting RDF [=Full=] should make sure to apply the reverse transformation
      to any graph that is known or likely to have been [=classicized=],
      to avoid creating such "hybrid" graphs.
```
— would benefit from rewriting to make it a single caution that covers both `[=Classic=]` applications classicizing, and `[=Full=]` applications "de-classicing" (there must be a better word! "the reverse transformation" doesn't cut it...), every graph


```suggestion
    <p>Another consequence of this restriction is that users will need to be aware and careful when merging graphs in an application that [=classicizes=] graphs or datasets.
      The concern is that merging a [=Full=] [=RDF graph=] containing at least one [=triple term=]
      with a [=classicized=] [=RDF graph=] (which might contain [=blank node=] instances of `rdf:TripleTerm`)
      could result in a "hybrid" graph that cannot be transformed to a consistent [=Full=] nor [=Classic=] [=RDF graph=].
      Therefor, such applications should [=classicize=] every graph prior to merging them.
      Conversely, applications supporting RDF [=Full=] should make sure to apply the reverse transformation
      to any graph that is known or likely to have been [=classicized=],
      to avoid creating such "hybrid" graphs.
      Since these transformations are designed to be <em>idempotent</em>, there is no harm in applying them more than necessary.
```